pkg/utils/mesh_config.go

```diff
@@ -116,6 +117,15 @@ func GetIngressGatewayCertValidityPeriod(mc v1alpha2.MeshConfig) time.Duration {
 		return defaultServiceCertValidityDuration
```

Comment:
might as well fix this here, looks like this got copy-pasted incorrectly and should be defaultIngressGatewayCertValidityDuration